src/ui-kit/experimental/hierarchical/hierarchical/hierarchical.component.spec.ts

```diff
@@ -55,19 +55,58 @@ describe('SamHierarchicalComponent', () => {
     ];
     component.configuration.childCountField = "childCount";
     component.configuration.topLevelBreadcrumbText = "All Departments";
-
-
     fixture.detectChanges();
   });
 
   it('should create', () => {
     expect(component).toBeTruthy();
   });
 
-
   it('single mode', () => {
     expect(component.isSingleMode()).toBe(false);
     component.model.treeMode = TreeMode.SINGLE;
     expect(component.isSingleMode()).toBeTruthy();
   });
+
+  it('single mode var', () => {
```

Comment:
@thomastighe Can you write more descriptive (English-y) test names? If I broke your test, I'm not sure I would be able to figure out what the intent of the test is in order to fix my code.

Try to keep the format `it("Should <do something> when <action happens> given <conditions or environment>`. In this case, `it("Should change to multiple mode when the model is updated and given a value of multiple")`

Reply:
Updated